### ISSUE TITLE
What is meant by this objective function ? is it conditional DCGAN ?

### ISSUE THREAD

**Issue by maryam089** [carpedm20/DCGAN-tensorflow#286]
def sigmoid_cross_entropy_with_logits(x, y):
      try:
        return tf.nn.sigmoid_cross_entropy_with_logits(logits=x, labels=y)
      except:
        return tf.nn.sigmoid_cross_entropy_with_logits(logits=x, targets=y)

self.d_loss_real = tf.reduce_mean( sigmoid_cross_entropy_with_logits(self.D_logits, tf.ones_like(self.D)))
self.d_loss_fake = tf.reduce_mean( sigmoid_cross_entropy_with_logits(self.D_logits_, tf.zeros_like(self.D_)))
self.g_loss = tf.reduce_mean( sigmoid_cross_entropy_with_logits(self.D_logits_, tf.ones_like(self.D_)))

**Comment by youngleec** [comment]
The try-except is to make sure that the code works with different versions of tensorflow. Whether it is conditional depends on whether using labels with images as inputs to discriminator and generator.

Hope it helps.

**Comment by maryam089** [comment]
`min
G
max
D
V (D, G) = Ex∼pdata(x)
[logD(x|y)]
+ Ez∼pz(z)
[log(1 − D(G(z|y)))]`
